components/table/table-wrapper.js

```diff
@@ -368,6 +380,10 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 			r.classList.toggle('d2l-table-selected-first', firstNonHeaderRow && isSelected);
 
 			Array.from(r.cells).forEach((c, index) => {
+				const isSortableCell = Array.from(c.childNodes).find((element) => element.localName === 'd2l-table-col-sort-button');
+				if (isSortableCell) {
+					c.style.padding = 0;
+				}
```

Comment:
I believe this should only be run if `isHeader` is true. We can also likely follow the same pattern here and toggle a class (e.g., `d2l-table-header-col-sortable`) and then add the style in the css. If we do that, we can put setting the padding in the css next to or near the place where the padding was originally set, making it more readable as to where it's being set and why. Using a class will also allow us to add other css that depends on that class if the need comes up.